.github/workflows/build-web-extension.yml

```diff
@@ -36,10 +36,14 @@ jobs:
             - name: Build
               env:
                   PL_SERVER_URL: ${{ secrets.PL_SERVER_URL }}
-              run: npm run web-extension:build
+              run: |
+                  node update-manifest-build.js
+                  npm run web-extension:build
             - name: Sign for Firefox
-              run: cd packages/extension/dist && web-ext sign --channel=unlisted --api-key=${{ secrets.PL_WEB_EXTENSION_FIREFOX_API_KEY }} --api-secret=${{ secrets.PL_WEB_EXTENSION_FIREFOX_API_SECRET }}
+              if: github.event.inputs.environment == 'Beta' || github.event.inputs.environment == 'Production'
+              run: cd packages/extension/dist && web-ext sign --id="{9e5ca7e8-2073-49d1-a93f-ac2f2088ff14}" --channel=unlisted --api-key=${{ secrets.PL_WEB_EXTENSION_FIREFOX_API_KEY }} --api-secret=${{ secrets.PL_WEB_EXTENSION_FIREFOX_API_SECRET }}
```

Comment:
I considered adding the ID to a secret, but this is publicly available and easily visible. It's also potentially better for someone searching for "where's an id from" to end up in this repo, I think. It can potentially be moved to a "visible env var" once we decide on something for #360